packages/dropdowns/src/elements/Trigger/Trigger.tsx

```diff
@@ -13,7 +13,7 @@ import { StyledInput } from '../../styled';
 import useDropdownContext from '../../utils/useDropdownContext';
 
 interface ITriggerProps extends HTMLAttributes<HTMLElement> {
-  /** Used to pass the ref callback to components with non-standard ref props i.e. `innerRef` */
+  /** Reference key for callback to components using non-standard ref props. i.e. `innerRef` */
```

Comment:
Per [content guidelines](https://garden.zendesk.com/content/word-list#ie) please reframe to get rid of the `i.e.`. Also, please restructure overall to follow RFC grammar guidelines, if possible. The preposition load is a bit much. Something like:

> Provides a reference key for trigger components that use a non-standard `ref` prop such as `innerRef`

Reply:
Fixed in latest push.